lib/debug/asserts.js

```diff
@@ -43,8 +43,23 @@ shaka.asserts.notImplemented = function() {};
 shaka.asserts.unreachable = function() {};
 
 
+shaka.asserts.patchAssert_ = function() {
+  var assert = console['assert'];
+
+  if (!assert) {
+    console.assert = function() {};
+  } else if (!assert.bind) {
```

Comment:
A comment here about what platforms/browsers do not have console.assert.bind would be helpful.